src/main/java/com/snack/news/repository/PicksRepositoryImpl.java

```diff
@@ -6,10 +6,7 @@
 import javax.persistence.EntityManager;
 import javax.persistence.PersistenceContext;
 import javax.persistence.TypedQuery;
-import javax.persistence.criteria.CriteriaBuilder;
-import javax.persistence.criteria.CriteriaQuery;
-import javax.persistence.criteria.Predicate;
-import javax.persistence.criteria.Root;
+import javax.persistence.criteria.*;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'javax.persistence.criteria'](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=4991072)